src/App.tsx

```diff
@@ -1,61 +1,117 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { getUserById } from './services/user';
+
+const todosWithUserId: Todo[] = todosFromServer.map(todo => {
+  const user = getUserById(todo.userId);
+
+  return { ...todo, user };
+});
```

Comment:
The `getUserById` function can return `null` if the user is not found. Ensure that the `user` property in the `Todo` type can handle `null` values, or handle the case where `getUserById` returns `null` to avoid potential runtime errors.